pkg/cmd/server/api/helpers.go

```diff
@@ -305,6 +305,7 @@ func GetKubeClient(kubeConfigFile string) (*kclient.Client, *restclient.Config,
 	// TODO: this should be configured by the caller, not in this method.
 	kubeConfig.QPS = 100.0
 	kubeConfig.Burst = 200
+	kubeConfig.ContentType = "application/vnd.kubernetes.protobuf"
```

Comment:
Will add the following:

1. New config flag that defaults to `application/json` for clients
2. When we write a config, we set `application/protobuf`
3. Ansible would have to manage the transition itself
4. The flag allows the "oh no proto is broken" revert on a release